amazonorders/selectors.py

```diff
@@ -95,3 +97,32 @@ class Selectors:
 
     FIELD_SELLER_NAME_SELECTOR = ["a", "span"]
     FIELD_SELLER_LINK_SELECTOR = "a"
+
+    #####################################
+    # CSS selectors for Transaction fields
+    #####################################
+
+    TRANSACTION_HISTORY_FORM_SELECTOR = f"form[method='post'][action$='{Constants.TRANSACTION_HISTORY_LANDING_ROUTE}']"
```

Comment:
Could we find a more dynamic way to select this that doesn't require a value in `Constants`? We should be able to select it by the suffix withou the URI.